tango/__main__.py

```diff
@@ -285,6 +285,14 @@ def cleanup(*args, **kwargs):
     type=str,
     help="""Specify the name for this run.""",
 )
+@click.option(
+    "-e",
+    "--ext-var",
+    type=str,
+    help="""JSONNET external variables to use when loading the experiment config.
+    For example, --ext-var 'pretrained_model=gpt2'.""",
+    multiple=True,
+)
```

Comment:
This is fine, but we could also import all environment variables into the Jsonnet context, right?

Can we make this a little more concise? `-D` for setting the variable, just like a C compiler? I don't care much about the long name, but `--ext-var` only makes sense when you know how Jsonnet thinks about the world, which is not something we should bother users with if we don't have to.

Reply:
> This is fine, but we could also import all environment variables into the Jsonnet context, right?

We already do this.

> Can we make this a little more concise?

Done: ec656cc

Reply:
But torchmetrics tries to import a bunch of dependencies that are not listed as dependencies, so it could be any of those.